src/imputation/MoR.py

```diff
@@ -240,13 +240,13 @@ def calculate_growth_rates(current_df, prev_df, target_vars):
     prev_df = (
         prev_df[["reference", "imp_class", "imp_marker"] + target_vars]
         .groupby(["reference", "imp_class"])
-        .sum()
+        .sum(numeric_only=True)
     ).reset_index()
 
```

Comment:
I say this because I think it's the index that changes, and Dylan found this caused an unexpected bug further down the pipeline. So I want to test it thoroughly rather than just seeing if the numbers initially look the same.